ecommerce-ui/src/tests/Rating.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Rating from '../Rental/components/Rating.js';
+import {shallow} from 'enzyme';
+import airbnbs from '../airbnbs.json';
+
+
+//import testData from './testData.json';
+
+// it('renders without crashing', () => {
+//   const div = document.createElement('div');
+//   ReactDOM.render(<Image />, div);
+//   ReactDOM.unmountComponentAtNode(div);
+// });
+
+it('should match the snapshot', () => {
+  const wrapper = shallow(<Rating rental={airbnbs[0]} />);
```

Comment:
Be sure to test the other props by passing dummy data or functions. (onClick, onBookRental)